packages/server-wallet/src/wallet/__test__/fixtures/test-channel.ts

```diff
@@ -98,7 +98,7 @@ export class TestChannel {
 
     return {
       ...this.channelConstants,
-      appData: '0x',
+      appData: '0x00',
```

Comment:
I'm confused by this in other places also — what's the right thing to be using here and what makes it right or wrong?

Reply:
Is the right thing to avoid magic strings, and have either a `ZERO_APP_DATA` or a `BYTES_ZERO` constant?

Reply:
I'll try and clear this up in another PR.